addon/components/markdown-to-html.js

```diff
@@ -25,7 +25,7 @@ const ShowdownComponent = Ember.Component.extend({
       }
     }
 
-    return new Handlebars.SafeString(this.converter.makeHtml(get(this, 'markdown')));
+    return Ember.String.htmlSafeg(this.converter.makeHtml(get(this, 'markdown')));
```

Comment:
Fixing the typo here should get the tests to pass. Needs to be `Ember.String.htmlSafe`